packages/dashboard/components/company-logo.tsx

```diff
@@ -7,10 +7,12 @@ import { useRouter } from 'next/router';
 export const CompanyLogo = ({ sx }: any) => {
   const theme = useTheme();
   const router = useRouter();
+
   const logo =
     theme.palette.mode === 'dark'
-      ? '/assets/logo-with-text.white.svg'
-      : '/assets/logo-with-text.purple.svg';
+      ? process.env.NEXT_PUBLIC_LOGO_LIGHT || '/assets/logo-with-text.white.svg'
```

Comment:
This is where this should be set:

https://github.com/transmute-industries/sidetree.js/blob/main/packages/dashboard/config/index.ts